### ISSUE TITLE
Anchor links with Chinese doesn't work.Toc content with other languages does not work

### ISSUE THREAD

**Issue by Phoenix-Ann** [victoriadrake/hugo-theme-introduction#108]
But there is another little bug,when content is English,it works.while content is Chinese it does't work.
When click Chinese content,it report as below
![image](https://user-images.githubusercontent.com/17487620/48833775-1b712c80-edb7-11e8-9636-4f2c35b9653e.png)

_Originally posted by @Phoenix-Ann in https://github.com/vickylai/hugo-theme-introduction/issues/32#issuecomment-440593221_

**Comment by hanzei** [comment]
Thanks for the report @Phoenix-Ann! Could you share a link to you web site, so we can reproduce the bug?

**Comment by Phoenix-Ann** [comment]
> Thanks for the report @Phoenix-Ann! Could you share a link to you web site, so we can reproduce the bug?

https://anna3.pub/blog/%E5%9C%A8windows%E4%B8%8A%E5%85%BC%E5%AE%B9python3%E5%92%8Cpython2/

**Comment by Phoenix-Ann** [comment]
I find when I set navbar-item in Chinese,it have the same performance.
![image](https://user-images.githubusercontent.com/17487620/49137199-b6718580-f326-11e8-8a2f-cb524ab350dd.png)

**Comment by Phoenix-Ann** [comment]
Oh~！ I find a way to resolve this bug.
Reason:

[> When using the address bar to submit query parameters, if not encoded, non-English characters will be encoded into the server according to the operating system's character set, and the server will decode according to the configured character set, so if the two are inconsistent, it will lead to garbled characters.
> 
> The encodeURI function uses UTF-8 to encode the URL, so if the server uses other encoding methods for decoding, it will be garbled. The default server configuration decoding character set is not UTF-8, so in most cases the address When the column submits Chinese query parameters, it will generate garbled characters; for this case, you can use the encodeURI twice to encode non-English characters on the client (mainly the browser).](https://blog.csdn.net/u012027337/article/details/52126475/)

Bug Fix:
File: hugo-theme-introduction/assets/js/index.js
line 27
`$("#nav-menu").removeClass("is-active");`
change to
`    $ encodeURI(encodeURI(("#nav-menu")).removeClass("is-active");`

Toc content  works both in Chrome,Firefox and IE

- **But navbar-item in Chinese just works in Chrome,not in Firefox and IE**

**Comment by hitzhangjie** [comment]
> $ encodeURI(encodeURI(("#nav-menu")).removeClass("is-active");

I tried this, it not works.

@hanzei I check the newest version, this problem still exists.

**Comment by hitzhangjie** [comment]
Hi, @hanzei, I fixed it.

```js
   // Bitty scrolling links script
   $("a[href^=\"#\"]").click(function(e) {
       e.preventDefault();
       //BUG fix: https://github.com/victoriadrake/hugo-theme-introduction/issues/108
       //
       //let's get the right attribute `id` of the heading
       v = e.target.href.replace(/http.*#/, '')
       id = decodeURI(v)
   
       $("html, body").animate({
           //scrollTop: $(document.getElementById(this.hash.substr(1))).offset().top                                                    
           scrollTop: $(document.getElementById(id)).offset().top
       }, 500);
       $("#nav-menu").removeClass("is-active");
       return true;
   })
```

**Comment by victoriadrake** [comment]
Resolved in #228
